smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SmithyModelFileInfoGenerator.kt

```diff
@@ -0,0 +1,25 @@
+package software.amazon.smithy.swift.codegen
+
+import software.amazon.smithy.aws.traits.ServiceTrait
+import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
+import software.amazon.smithy.swift.codegen.model.getTrait
+
```

Comment:
This file generates a file named `smithy-model-info.json` into a service client target.

Its contents are
```json
{"path":"relative/path/to/model.json"}
```
where `relative/path/to/model.json` is the path to the JSON AST file for this service, relative to project root.

The Swift code generator will use this file to locate the model at build time.